Assets/MixedRealityToolkit/UX/Scripts/Dialog/DialogButton.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using MixedRealityToolkit.UX.Buttons.Utilities;
+using MixedRealityToolkit.UX.Dialog;
+using UnityEngine;
+
+namespace MixedRealityToolkit.UX.Buttons
+{
+    public class DialogButton : MonoBehaviour
+    {
+        public DialogShell ParentDialog { get; set; }
```

Comment:
Is this value expected to be set via inspector? If so we need a serialized private backing field.